src/Microsoft.Azure.WebJobs.Host/Bindings/BindingFactory.cs

```diff
@@ -55,7 +68,7 @@ public IConverterManager ConverterManager
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1822:MarkMembersAsStatic")]
         public IBindingProvider AddTypeFilter(Func<Type, bool> predicate, IBindingProvider innerRule)
```

Comment:
Do we still need AddTypeFilter and AddValidator? I was hoping we could go from 3 'filter' methods to 1 and reduce the concept count.